recipes/hail/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "hail" %}
+{% set version = "0.2.34" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: 9c3a4e5f81ad23b569c1d6dc80bfb070
+  
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```

Comment:
Please add the `entry_point` for the command line tools that are listed under `console_scripts` in the `setup.py`. See:

https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#python-entry-points